torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -29,9 +35,30 @@ def __init__(
     ):
         super().__init__()
 
-        self.attribution_method = attribution_method
+        import captum.attr  # noqa
+
         self.kwargs = kwargs
 
+        if isinstance(attribution_method, str):
+            self.attribution_method = getattr(
+                captum.attr,
+                attribution_method,
+            )
+        else:
+            self.attribution_method = attribution_method
+
```

Comment:
```suggestion
def supports():
    return True
```
It looks like all configs are supported. Except for probably a node mask `object`.

Reply:
`common_attributes` and `object` are not supported.